tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Requests/DeleteOrderRequest.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Threading.Tasks;
+using NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Responses;
+using NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Steps.Common;
+using NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Utils;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Requests
+{
+    internal sealed class DeleteOrderRequest
+    {
+        private readonly Request _request;
+        private readonly string _deleteOrderUrl;
+
+        public DeleteOrderRequest(
+            Request request,
+            string orderingApiBaseAddress,
+            string orderId)
+        {
+            _request = request ?? throw new ArgumentNullException(nameof(request));
+            OrderId = orderId ?? throw new ArgumentNullException(nameof(orderId));
+
+            _deleteOrderUrl =
+                $"{orderingApiBaseAddress}/api/v1/orders/{orderId}";
+        }
+
+        public string OrderId { get; }
+
+        public async Task<Response> ExecuteAsync()
+        {
+            var response = await _request.DeleteAsync(_deleteOrderUrl);
```

Comment:
Very minor, could just return rather than introduce a variable.

Reply:
ah yes I meant to do this! 👍